### DISCUSSION TITLE
Settings sync button in github.dev editor does nothing.

### DISCUSSION THREAD

**Discussion by joseantoniocb** [community/community#21162] [Projects and Issues]
When I edit a file in the web editor everything works fine expect the Settings Sync button, when I click on it nothing happens. 

I'm able to sync my settings on my Mac, my PC, and with vscode.dev except on the github.dev web editor.

(I´ve tried to sync my settings in Safari, Edge and Chrome even in incognito mode with no luck)

Can anyone help me or guide me with this issue?

Thanks in advance and my best regards.

https://user-images.githubusercontent.com/1056477/180627722-96fa16a7-db52-4073-89b6-423662588268.mov

**Comment by wneel** [comment]
**You have to be connected to turn on Sync settings**,
So just continue and then you are connected, click on the "profile icon" at the bottom left and here turn on sync settings,

i had the same issue and it worked for me 
![image](https://user-images.githubusercontent.com/66263633/180628668-2932a10c-7522-4777-b211-abdeea9ed763.png)

**Comment by joseantoniocb** [comment]
@x-Wawa thanks for your reply but I don't have the Settings Sync option on my Profile icon.

https://user-images.githubusercontent.com/1056477/180628803-9079ce70-e9ad-4777-8319-e55583d8b8b6.mov

**Reply by wneel** [reply]
oh mb, and can you access to this settings, toggle them ? 

![image](https://user-images.githubusercontent.com/66263633/180629071-dd020ba3-e4b4-423d-beb8-5202f89063dc.png)
![image](https://user-images.githubusercontent.com/66263633/180629072-820d1034-6559-4466-9bcc-936e9315794b.png)

**Reply by wneel** [reply]
btw do you get error or something printed out in the console when pressing the button ? or a request would be catched

**Reply by joseantoniocb** [reply]
No, no Settings sync options. its weird because everything works fine on VS Code PC/Mac and with vscode.dev the issue is only present in the github.dev web editor. I'm going nuts!!!

![Screen Shot 2022-07-23 at 21 12 47](https://user-images.githubusercontent.com/1056477/180629312-ae13bd14-0db6-4db9-8610-ce531d03e9b9.png)

**Reply by joseantoniocb** [reply]
BTW. Thanks for your time, I really appreciate it 🙏

**Reply by wneel** [reply]
yeah, i try as i can, last thing does sync settings on https://vscode.dev/ work for you or it's just the GitHub malediction
